docs/blog/posts/argocd-keptn-health.md

```diff
@@ -0,0 +1,180 @@
+which has a healthy green status in ArgoCD UI, but it's not working as expected
+due to a slow `response time` of the application.
+
+We will deploy a simple `podtato-head` application, which consists of multiple
+Deployments and Services via ArgoCD.
+The Argo Application deploying the manifests can look like the following:
+
+```yaml
+{% include "./argocd-keptn-health/argo-app.yaml" %}
+```
+
+After a few moments, the `podtato-head` application is successfully deployed and all pods
+are running.
+
+![Running Pods](./argocd-keptn-health/running-pods.png)
+
+We can also examine the ArgoCD UI and everything seems to be working as expected and the
+`podtato-head` application is healthy.
+
+![Healthy App](./argocd-keptn-health/healthy-app.png)
+
+Let's now try to add some health checks of the `podtato-head` application
+and use Keptn to execute them.
+For this, we are going to use the
+[Keptn Release Lifecycle Management](https://keptn.sh/stable/docs/getting-started/lifecycle-management/)
+feature and perform the checks via `KeptnEvaluations`.
+For this example, we assume that you already have a metrics provider deployed
+and configured on your cluster, which is able to fetch the `response time` values
+of the microservices.
+In our setup, we are going to use [Prometheus](https://prometheus.io/).
+
+First, we need to create [KeptnMetric](https://keptn.sh/stable/docs/reference/crd-reference/metric/)
+and [KeptnMetricsProvider](https://keptn.sh/stable/docs/reference/crd-reference/metricsprovider/)
+resources in our cluster.
+These two resources contain a simple query for fetching `response time` of the `podtato-head`
+application microservice as well as configuration for the metrics provider supplying the data.
+
+```yaml
+{% include "./argocd-keptn-health/metric.yaml" %}
+```
+
+Next, we add `KeptnEvaluationDefinition` into our git repository, where our
+`podtato-head` application lives.
+It defines the [SLO](https://www.dynatrace.com/news/blog/what-are-slos/)
+by linking the existing `KeptnMetric` resource and providing the rule the value should fullfil.
+
+```yaml
+{% include "./argocd-keptn-health/evaluationdefinition.yaml" %}
+```
+
+Additionally, we annotate the `podtato-head-frontend` Deployment to execute
+the task as part of `post-deployment-evaluation` checks.
+
+```yaml
+{% include "./argocd-keptn-health/annotation.yaml" %}
+```
+
+After these two changes are made in our git repository, ArgoCD will see changes and re-trigger
+the deployment of `podtato-head`.
+Keptn waits until all of the
+application pods are running and afterwards, it executes `post-deployment-evaluation` tasks.
+
+Due to slow `response time` of the `podtato-head-frontend` microservice, the
+executed `KeptnEvaluation` fails.
+
+Here we can see that with the use of Keptn we can perform more advanced health checks
+(tasks or evaluations) and verify that the application is healthy during the process
+of deployment which is performed by ArgoCD.
```

Comment:
```suggestion
(tasks or evaluations) and verify that the deployed application is healthy,
to supplement the ArgoCD verification that the deployment is healthy.
```
This may need more rework.  But the current text says that Keptn is veryifying the ArgoCD "process of deployment" and I don't think that is what you mean.